Base/usr/share/man/man1/profile.md

```diff
@@ -1,24 +1,44 @@
 ## Name
 
-profile
+profile - Process or system profiler
 
 ## Synopsis
 
-```sh
-$ profile [-p PID] [-a] [-e] [-d] [-f] [-w] [-c command] [-t event_type]
+```**sh
+$ profile [-p PID] [-a] [-e] [-d] [-f] [-w] [-t event_type] [COMMAND_TO_PROFILE]
 ```
 
-## Options:
+## Description
+
+`profile` records profiling information that can then be read with `ProfileViewer`.
+
+## Options
 
 * `-p PID`: Target PID
 * `-a`: Profile all processes (super-user only), result at /sys/kernel/profile
 * `-e`: Enable
 * `-d`: Disable
 * `-f`: Free the profiling buffer for the associated process(es).
 * `-w`: Enable profiling and wait for user input to disable.
-* `-c command`: Command
 * `-t event_type`: Enable tracking specific event type
 
 Event type can be one of: sample, context_switch, page_fault, syscall, read, kmalloc and kfree.
 
-<!-- Auto-generated through ArgsParser -->
```

Comment:
I believe changes like this should also remove the application from a list in one of the Meta/ files that runs in the "check man pages" job?

Reply:
Good catch! I assumed that was automatic, not hard-coded. :^(

I found `/Base/root/generate_manpages.sh`, presumably that's the one.

Reply:
Yep that's what I've seen changed before.